src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -52,6 +52,7 @@ public static class MetricEventNames
         // function level events
         public const string FunctionInvokeLatency = "function.invoke.latency";
         public const string FunctionBindingTypeFormat = "function.binding.{0}";
+        public const string BindToParameterBindingData = "function{0}.binding.parameterbindingdata";
```

Comment:
The format parameter is the function name? I think this metric name should just be function.binding.parameterbindingdata or maybe better function.binding.deferred, and the function name is handled by the LogEvent overload that takes a function name as a parameter.

Reply:
The name function.binding.deferred looks good. Updated.